examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
I think this is also related to what @Eric89GXL was talking about. In the old way, it would first pick data using `picks` then it would take rows/columns from the projection matrix according to picks, `np.dot(proj[picks][:, picks], data)`. In this case, it's taking a matrix of shape `(n_picks, n_picks)` and doing dot product with our data of shape `(n_picks, n_times)`. In this case, assuming that channel information is shared between rows/columns of the projection matrix, we'd only be including those rows/columns that had a notch filter applied.

When we use `apply_proj`, we don't do any picking, so we dot w/ our projection matrix of shape `(n_channels, n_channels)` by our data `(n_channels, n_times)`. So in this case, when the mixing matrix is applied it's using information from all channels, including those that didn't have a notch filter applied.

Thoughts on what's the next best step here? I'm not sure which of these two cases is the preferred one, since I don't know anything about projection vectors (though this PR is teaching me quite a bit haha).